### PR TITLE
Add meta viewport for proper responsive scaling.

### PR DESCRIPTION
**Pull request by XhmikosR**
/CC @fmarier @mozfreddyb

### CODE REVIEW

**Comment by XhmikosR** [comment]
ping @fmarier @mozfreddyb

**Comment by mozfreddyb** [comment]
Sorry, I've been out of office for the last month and just came back yesterday. This will have to wait some more.

**Comment by XhmikosR** [comment]
Np, @mozfreddyb, as long as it's not hanging here for months :)

PS. Should I close #164 and just push everything here? I'm personally using the npm-scripts branch and works perfectly fine apart from the missing copyright functionality checking; we could do it with a simple script later if needed.

**Comment by mozfreddyb** [comment]
Thank you for putting so much work into srihas.org, but it's unlikely we can merge this PR as it is :(

I actually have a very hard time following so many commits in so many pull requests. It's hard to review code changes that come together with indentation/style changes, splitting these out would be a bare minimum.
But even if each of those commits were in its own PR, I still have a hard time seeing the benefit of all those changes. I know redoing all of this work isn't very exciting and it feels like this needs planning and a better conversation before we should move on.

That all being said, @fmarier and I don't have a lot of time on our hands to work on SRI, so scheduling this won't be easy either :/

**Comment by XhmikosR** [comment]
Well, I can split some of them, but it's not that hard; just review each commit separately...

Really, the changes are self-explanatory. And I'd hate not to get these merged, because like I said, it's pretty simple if you look at each commit.

File: templates/hash.html [hunk 1]
```diff
@@ -5,7 +5,7 @@
 <html lang="en">
   <head>
     <meta charset="utf-8">
-    <meta http-equiv="Referrer-Policy" content="no-referrer, strict-origin-when-cross-origin">
+    <meta name="referrer" content="no-referrer, strict-origin-when-cross-origin">
```
Comment (XhmikosR) [inline templates/hash.html]: @mozfreddyb: for example this was an error; there's no `http-equiv="Referrer-Policy"`

**Pull request merged**